src/RageThreads.cpp

```diff
@@ -682,12 +682,12 @@ void LockMutex::Unlock()
 
 	mutex.Unlock();
 
-	constexpr uint_fast64_t THRESHOLD_USEC = 15000;
+	constexpr uint64_t THRESHOLD_USEC = 15000;
 
-	if (file && locked_at != FAST_ULL_MAX)
+	if (file && locked_at != UINT64_MAX)
```

Comment:
Where is UINT64_MAX defined?

Reply:
If anything I think w can make use of numeric_limits

Reply:
> Where is UINT64_MAX defined?

IT is part of `stdint.h` -  happy to use numeric_limits instead though